test/evergreen/import_compatibility_test.sh

```diff
@@ -21,11 +21,20 @@ build_branch()
 
     git checkout --quiet "$1"
 
-    config=""
-    config+="--enable-snappy "
-    config+="--disable-standalone-build "
-    (sh build_posix/reconf &&
-         ./configure $config && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
+    if [ "${build_sys[$1]}" == "cmake" ]; then
+            . ./test/evergreen/find_cmake.sh
+            config=""
+            config+="-DENABLE_SNAPPY=1 "
+            config+="-DWT_STANDALONE_BUILD=0 "
+            (mkdir build && cd build &&
+                $CMAKE $config ../. && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
```

Comment:
Why don't we use `ninja` if possible?

Reply:
Generally I do encourage building with `ninja`, but I opted with `make` here to be consistent with the `autoconf` paths. `Ninja` is not available on our our build machines (i.e. rhel, macos), where `make` is generally pretty available and standard across the board. I wanted to keep the dependencies of this script simple, making it as portable as possible (as it could be run manually outside evergreen). 
Whilst it might be possible to test for `ninja` and then fallback to `make`, it feel it over complicates the script, where just using `make` keeps the script simple and consistent.

Reply:
Can we ask the BUILD team to install ninja?

Reply:
We can definitely do that, it would nice if ninja was installed on those machines. 
However I feel that's outside the scope of this script and ticket, since Make works pretty well here. I rather not make that a blocker here.

Reply:
I agree and created WT-8574. Can you add a FIXME-WT-8574 comment?